.changeset/seven-jars-yell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@graphql-inspector/core': major
```

Comment:
I could be convinced that this is a minor patch because the changes to the output's format doesnt break anything. But the actual content of the changes has changed drastically which is why I thought we should be safe and declare a major change.

Reply:
> we need to see how we can introduce this backwards-compatible into Hive Console

Agreed. My strategy is to make the new fields optional in Hive Console. If I'm careful about it, this should let us safely migrate without breaking any existing change logic.

Reply:
I've added a much more detailed changelog message but didn't go to the length of listing every single field being changed.
If there's a case for listing every single field being changed, then I can do that. Or maybe it would be better suited for a migration guide?